src/components/GraphFilter/LayoutDropdown.tsx

```diff
@@ -20,33 +21,49 @@ export class LayoutDropdown extends React.Component<LayoutDropdownProps, LayoutD
     ['dagre', 'Dagre'],
     ['klay', 'Klay']
   ];
+  static InitialLabel = 'Select Graph Type';
+
   constructor(props: LayoutDropdownProps) {
     super(props);
     this.state = {
-      currentGraphType: props.initialLayout
+      currentGraphType: props.initialLayout,
+      currentGraphTypeLabel: this.getGraphTypeName(props.initialLayout)
     };
   }
 
+  getGraphTypeName = (graphType: string) => {
```

Comment:
Same as above: lookup function can be avoided if you pass the label+value as event key.